l2geth/eth/filters/api.go

```diff
@@ -52,24 +52,22 @@ type filter struct {
 // PublicFilterAPI offers support to create and manage filters. This will allow external clients to retrieve various
 // information related to the Ethereum protocol such als blocks, transactions and logs.
 type PublicFilterAPI struct {
-	backend    Backend
-	mux        *event.TypeMux
-	quit       chan struct{}
-	chainDb    ethdb.Database
-	events     *EventSystem
-	filtersMu  sync.Mutex
-	filters    map[rpc.ID]*filter
-	rangeLimit bool
```

Comment:
@InoMurko added this for the `eth_getLogs`. Probably we should keep this?

Reply:
Yup!